RawWaterManagement/OpenChannelSystem/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,94 @@
+{
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -82.1141235,
+        -159.124903
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Young military guy nearly dinner. Course fear certainly reveal while truth. Thousand reduce ready.",
+      "addressLocality": "Federal or we business. Peace body bank seem idea.",
+      "addressRegion": "Impact national respond as like up significant. Officer wonder inside worry property response himself choice.",
+      "addressCountry": "Lead artist summer site newspaper century land. Course very tough business. Foot full blood stuff behind.",
+      "postalCode": "Service easy sure best throw. Budget experience run past go sister. Table thing best.",
+      "postOfficeBoxNumber": "Be candidate buy traditional air word herself product. High share parent stock almost. Success poor source might.",
+      "areaServed": "Both example finally fish year evening power local. Control fire management left. Almost her ground choose condition say player. Go my model stock let prepare."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Cup bar series assume majority defense that by. Many manager somebody approach. Lay leg page summer line."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:NXDF:16687280",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2003-07-15T09:19:21Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1992-11-16T21:11:30Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Increase film these such cover. Analysis total woman or official successful money."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Lot section establish most. Movie important white himself somebody. Within buy once land energy avoid."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Space into politics rich compare international certainly. Television its place just let."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Leave finally may last mission. Day by laugh test former. Artist sort process shake lead when year central. Age until democratic major."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Three course understand. White week catch one."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:MKXR:87751348",
+      "urn:ngsi-ld:dataModel:items:BDXZ:05485215"
+    ]
+  },
+  "seeAlso": {
```

Comment:
Type suggested toi be relationship
1. Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph. 
2. Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array. 
3. Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId 

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
	{
	   "type":"Relationship",
	   "object":"urn:ngsi-ld:Room:01",
	   "datasetId":"urn:ngsi-ld:dataSet:01"},
	},
	"type":"Relationship",
	"object":"urn:ngsi-ld:Room:02" ,
	"datasetId":"urn:ngsi-ld:dataSet:02"}
	]